src/thunderbots/software/util/parameter/config/evaluation.yaml

```diff
@@ -0,0 +1,50 @@
+Evaluation:
```

Comment:
We should probably put this under the `AI:` section of `ai.yaml`, just to keep AI configuration in one place for now.

Reply:
`Evaluation:` is now moved under `AI:` and `evaluation.yaml` has been removed